loading-code/typescript-default-extenstion/src/helpers/planet-helpers.ts

```diff
@@ -0,0 +1,13 @@
+import { InterstellarObjects, NotRealPlanets, RealPlanets } from '../planets.js'
```

Comment:
The `.js` file extension is suspicious here - isn't this example doing "extensionless"?

```suggestion
import { InterstellarObjects, NotRealPlanets, RealPlanets } from '../planets'
```